src/components/nav_drawer/nav_drawer.tsx

```diff
@@ -177,14 +244,14 @@ export class EuiNavDrawer extends Component {
         // does not allow for deep `ref` element management at present
         const element = document.querySelector(
           `#${MENU_ELEMENT_ID} [${ATTR_SELECTOR}='${focusReturn}']`
-        );
+        ) as HTMLInputElement;
         if (!element) return;
         requestAnimationFrame(() => {
           element.setAttribute('aria-expanded', 'false');
         });
         if (!shouldReturnFocus) return;
         requestAnimationFrame(() => {
-          element.focus();
+          element!.focus();
```

Comment:
The `!` is unnecessary as the above `if (!element) return;` check handles this case.

```suggestion
          element.focus();
```